com.unity.render-pipelines.core/Runtime/Utilities/ScreenCoordOverrideUtils.cs

```diff
@@ -0,0 +1,29 @@
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// Screen Coord Override Utility class.
+    /// </summary>
+    public class ScreenCoordOverrideUtils
```

Comment:
Do you really need to create a new class to enable/disable keyword? looks like simply defining 
const string k_ShaderKeyword = "SCREEN_COORD_OVERRIDE"; will be sufficient
(This is a remark, I am not against it but seems extra code we could avoid.

Reply:
i.e referring to this code 

               if (m_ScreenCoordOverride)
                {
                    ScreenCoordOverrideUtils.EnableKeyword(passData.uberPostCS);
                }

                if (PostProcessEnableAlpha())
                {
                    passData.uberPostCS.EnableKeyword("ENABLE_ALPHA");
                    
                    
which can simply be rewrite

               if (m_ScreenCoordOverride)
                {
                     passData.uberPostCS.EnableKeyword(k_ScreenOverrideShaderKeyword);
                }

                if (PostProcessEnableAlpha())
                {
                    passData.uberPostCS.EnableKeyword("ENABLE_ALPHA");

Reply:
This is a very shallow class I initially added not to duplicate the hardcoded shader keyword but I'm removing it :)